prefs.py

```diff
@@ -60,17 +60,17 @@ class PrefsVersion:
     def __init__(self):
         return
 
-    def stringToSerial(self, versionStr: str) -> int:  # noqa: N802 # used in plugins
+    def stringToSerial(self, version_str: str) -> int:  # noqa: N802 # used in plugins
         """
```

Comment:
Cant rename the param; will break keyword args, unless you capture everything in `**kwargs` and check

Reply:
Despite the comment, this isn't documented in PLUGINS.md, so isn't actually a published interface.  I probably had a reason for the 'used in plugins' comment though.